src/site/content/en/angular/faster-angular-change-detection/index.md

```diff
@@ -171,12 +164,17 @@ You can read more about the `OnPush` change detection strategy in the [official
 
 To see the effect of this optimization, enter a new employee in the application below:
 
-<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
-  <iframe
-    src="https://stackblitz.com/github/mgechev/change-detection-web-dev/tree/onpush?embed=1&file=src/app/app.component.ts&view=preview"
-    style="height: 100%; width: 100%; border: 0;">
-  </iframe>
-</div>
+<figure class="w-figure w-figure--fullbleed">
```

Comment:
oh, sorry for any confusion. I was suggesting replacing _all_ of the videos with links to the Stackblitz projects.

Reply:
No worries. Just dropped the other 2 videos as well.